src/App.js

```diff
@@ -1,13 +1,71 @@
 import React from 'react';
-
 import './App.scss';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { GoodsList } from './GoodsList';
+
+class App extends React.Component {
+  state = {
+    show: false,
```

Comment:
You don't need show property here. 
When you press any button, you should immediately see the requested list.